packages/core/src/utils.ts

```diff
@@ -231,10 +232,11 @@ export function mapContext<
 >(
   mapper: Mapper<TContext, TEvent, any> | PropertyMapper<TContext, TEvent, any>,
   context: TContext,
-  event: TEvent
+  event: TEvent,
+  self: AnyActorRef
 ): any {
   if (typeof mapper === 'function') {
-    return mapper({ context, event });
+    return mapper({ context, event, self });
```

Comment:
`self` is not passed here to the property mapper (`subMapper`) that is called below